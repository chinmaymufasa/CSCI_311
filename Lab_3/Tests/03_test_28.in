2
145
5
2
76
5
2
79
5
2
187
5
2
58
5
2
44
5
2
134
5
2
80
5
2
113
5
2
118
5
2
118
5
2
176
5
2
104
5
2
38
5
2
38
5
2
16
5
2
39
5
2
61
5
2
93
5
2
184
5
1
20
-1