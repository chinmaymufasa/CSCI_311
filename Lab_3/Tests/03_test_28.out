145 
76 145 
76 79 145 
76 79 145 187 
58 76 79 145 187 
44 58 76 79 145 187 
44 58 76 79 134 145 187 
44 58 76 79 80 134 145 187 
44 58 76 79 80 113 134 145 187 
44 58 76 79 80 113 118 134 145 187 
44 58 76 79 80 113 118 118 134 145 187 
44 58 76 79 80 113 118 118 134 145 176 187 
44 58 76 79 80 104 113 118 118 134 145 176 187 
38 44 58 76 79 80 104 113 118 118 134 145 176 187 
38 38 44 58 76 79 80 104 113 118 118 134 145 176 187 
16 38 38 44 58 76 79 80 104 113 118 118 134 145 176 187 
16 38 38 39 44 58 76 79 80 104 113 118 118 134 145 176 187 
16 38 38 39 44 58 61 76 79 80 104 113 118 118 134 145 176 187 
16 38 38 39 44 58 61 76 79 80 93 104 113 118 118 134 145 176 187 
16 38 38 39 44 58 61 76 79 80 93 104 113 118 118 134 145 176 184 187 
Not found
